spec/features/admin/product_import_spec.rb

```diff
@@ -366,7 +366,7 @@
 
   def import_data
     expect(page).to have_selector 'a.button.proceed', visible: true
-    click_link I18n.t('admin.product_import.import.import')
+    within "#content" do click_link I18n.t('admin.product_import.import.import') end
```

Comment:
I'd vote for `{}` instead of `do end` for single-blocks. That's the Ruby's standard-ish. Strange that Rubocop didn't complain.

Reply:
Done.